tools/bump-version

```diff
@@ -11,12 +11,22 @@ program
   .requiredOption('--owner <owner>', 'owner of the current repository, usually "Integreat"')
   .requiredOption('--repo <repo>', 'the current repository, usually "integreat-webapp" or "integreat-react-native-app"')
   .requiredOption('--branch <branch>', 'the current branch')
+  .requiredOption('--release-notes <release-notes>', 'the release notes as stringified json')
+  .option('--development-release', 'whether the release is a development release which is not delivered to production')
 
-const commitAndTag = async (versionName, versionCode, { deliverinoPrivateKey, owner, repo, branch }) => {
+const commitAndTag = async (versionName, versionCodeString, {
+  deliverinoPrivateKey, owner, repo, branch, developmentRelease, releaseNotes
+}) => {
   const appOctokit = await authenticate({ deliverinoPrivateKey, owner, repo })
   const versionFileContent = await appOctokit.repos.getContent({ owner, repo, path: VERSION_FILE, ref: branch })
 
-  const contentBase64 = Buffer.from(JSON.stringify({ versionName, versionCode })).toString('base64')
+  const versionCode = versionCodeString ? parseInt(versionCodeString) : null
```

Comment:
I think this should be the previous value and not `null` or `undefined`.

Reply:
What is the previous value if there is no version code string? :thinking: